### ISSUE TITLE
Twilio Verify

### ISSUE THREAD

**Issue by enchorb** [RJPearson94/terraform-provider-twilio#51]
Hey this provider is awesome, much better than the one that comes directly from Twilio.  The one thing missing it seems is the Verify API which is in the 'official' one. Is it possible to add to this one as well, for reference: https://github.com/twilio/terraform-provider-twilio/tree/main/examples/verify/v2

**Comment by RJPearson94** [comment]
Hi @enchorb 

Thanks for raising this feature request. I will take a look at implementing the changes you have asked for, at this time I cannot give an ETA for this feature but I'll take a look at implementing it.

Are there any resources in particular that you would like added to the provider? In the example you shared I see it shows the verify service resource but do you need the provider to support webhooks, messaging configuration, service rate limiting, etc.?

Thank again for the feature request and any additional feedback on the provider would be greatly appreciated

**Comment by enchorb** [comment]
Hey messaging configuration would be great to have automated as well, we don't use the rate limiter although I'm sure many others may

**Comment by RJPearson94** [comment]
Hi @enchorb

The following resources and data sources have been added to the provider

- **New Resource:** `twilio_verify_messaging_configuration` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/resources/verify_messaging_configuration.md)
- **New Resource:** `twilio_verify_service` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/resources/verify_service.md)
- **New Resource:** `twilio_verify_service_rate_limit` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/resources/verify_service_rate_limit.md)
- **New Resource:** `twilio_verify_service_rate_limit_bucket` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/resources/verify_service_rate_limit_bucket.md)
- **New Resource:** `twilio_verify_webhook` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/resources/verify_webhook.md)
- **New Data Source:** `twilio_verify_messaging_configuration` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_messaging_configuration.md)
- **New Data Source:** `twilio_verify_messaging_configurations` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_messaging_configurations.md)
- **New Data Source:** `twilio_verify_service` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_service.md)
- **New Data Source:** `twilio_verify_service_rate_limit` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_service_rate_limit.md)
- **New Data Source:** `twilio_verify_service_rate_limits` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_service_rate_limits.md)
- **New Data Source:** `twilio_verify_service_rate_limit_bucket` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_service_rate_limit_bucket.md)
- **New Data Source:** `twilio_verify_service_rate_limit_buckets` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_service_rate_limit_buckets.md)
- **New Data Source:** `twilio_verify_webhook` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_webhook.md)
- **New Data Source:** `twilio_verify_webhooks` [docs](https://github.com/RJPearson94/terraform-provider-twilio/blob/main/docs/data-sources/verify_webhooks.md)


This has now been released as part of v0.18.0. This release is now available in the Terraform Registry.

Please update your Terraform configuration to include the following

```
terraform {
  required_providers {
    twilio = {
      source = "RJPearson94/twilio"
      version = "0.18.0"
    }
  }
}
```
